dashboard/src/components/DeleteButtonAndConfirmModal.tsx

```diff
@@ -50,7 +50,7 @@ const DeleteButtonAndConfirmModal = ({
       <ModalContainer open={open} onClose={() => setOpen(false)} size="3xl">
         <ModalHeader>
           <div className="tw-px-4">
-            <p className="tw-block tw-text-center tw-text-xl tw-text-red-500">{title}</p>
+            <div className="tw-block tw-text-center tw-text-xl tw-text-red-500">{title}</div>
```

Comment:
c'est mieux de ne pas avoir de marge, mais je doute que ce soit mieux d'avoir du texte dans une `div` au lieu d'un `p`

ici ils font du `h2` donc je propose la même chose https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role

```suggestion
            <h2 className="tw-block tw-text-center tw-text-xl tw-text-red-500">{title}</h2>
```